llama-rs/src/loader_common.rs

```diff
@@ -105,7 +105,7 @@ pub enum LoadProgress<'a> {
         /// The path to the model part.
         file: &'a Path,
         /// The number of bytes in the part.
-        byte_size: usize,
+        byte_size: u64,
```

Comment:
I'm open to reverting this, but it seems reasonable to match the type of https://doc.rust-lang.org/std/fs/struct.Metadata.html#method.len.